wolfcrypt/test/test.c

```diff
@@ -27625,6 +27625,20 @@ WOLFSSL_TEST_SUBROUTINE wc_test_ret_t hpke_test(void)
     #endif
 #endif
 
+/* added this code to hit code coverage on lines 219-225 on hpke.c*/
+#if defined(HAVE_CURVE448) && \
+    (defined(WOLFSSL_SHA384) || defined(WOLFSSL_SHA512))
+    /* test with_curve448 */
+    ret = wc_HpkeInit(hpke, DHKEM_X448_HKDF_SHA512, HKDF_SHA512,
+        HPKE_AES_128_GCM, NULL);
+
+    if (ret != BAD_FUNC_ARG) /* Curve448 is not supported yet */
+        return WC_TEST_RET_ENC_EC(ret);
+
+    if (ret == 0)
```

Comment:
You don't want to return here. Remove this. Also make sure and reset `ret = 0;` to avoid future confusion.

Reply:
For this are you saying to replace "return WC_TEST_ENC_EC(ret);"  with "ret = 0;" ?

Reply:
Your code above is fine. Don't return, continue execution and make sure `ret = 0;`.